README.md

```
@@ -107,6 +107,17 @@ This project plan ensures the creation of a robust, scalable, and user-friendly
 3. Write descriptive commit messages
 4. Create pull requests for review
 
+## API Development Guidelines
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)